app/views/families/show.html.erb

```diff
@@ -29,7 +29,7 @@
     </div>
   </div>
 
-<% if @family.loan %>
+<% if @family.loan && @family.loan.active? %>
 <h4>This family has requested a Microloan to begin a business of their own</h4>
 <table class="table">
   <tr>
```

Comment:
Looks good. At some point we should look into making sure we get a lot of these conditional statements out of our views.